shims/mapr510/assemblies/mapr510-shim/src/assembly/assembly.xml

```diff
@@ -47,6 +47,10 @@
         <include>org.apache.pig:pig</include>
         <include>org.apache.sqoop:sqoop</include>
         <include>com.twitter:parquet-hadoop-bundle</include>
+        <include>org.apache.avro:avro</include>
```

Comment:
avro, jackson libraries come from mapr client, so they shouldn't be packed for mapr shims, for compilation for mapr51 - 1.7.4, for mapr52-1.7.5(the version checked on mapr client or cluster)